README.md

```diff
@@ -1,4 +1,19 @@
-# fcli
-Foundational Components CLI
-
+ - Foundational Components CLI
```

Comment:
This looks weird when formatted.  It's a bullet point with the sentence on line two merged with this.  What was the intention?